bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -139,6 +139,20 @@ impl Room {
     pub fn remove_timeline(&self) {
         *self.timeline.write().unwrap() = None;
     }
+
+    pub fn retry_decryption(&self, session_ids: Vec<String>) {
```

Comment:
Presumably this only works for `megolmV1AesSha2` session IDs or can I pass a `olmV1Curve25519AesSha2` in here too?

Reply:
As just noted on Matrix, this is right. olm is for to-device messages (so really the enum variant for timeline items is actually pretty pointless).

Reply:
Great, let's please maybe remove that at some point 🙏